torch/distributed/fsdp/_init_utils.py

```diff
@@ -659,7 +668,9 @@ def _get_ignored_modules(
     for module in ignored_root_modules:
         if not isinstance(module, torch.nn.Module):
             raise TypeError(msg_prefix + f"but got an iterable with {type(module)}")
-        if isinstance(module, fsdp_file.FullyShardedDataParallel):
+        if isinstance(
+            module, fsdp_file.FullyShardedDataParallel
+        ) or _get_module_fsdp_state(module):
```

Comment:
Since this is only called once during initialization, I think we can simplify this to just:
```
if _get_module_fsdp_state(module):
```
?

I am thinking the reason is that for wrapper path, we still have `_get_module_fsdp_state(module) is not None`.